2023-10-24 15:33:06 - D:\Hub\Reptiles_code\github_reptiles.py[line:38] - ERROR: ('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2023-10-24 17:25:49 - D:\Hub\Reptiles_code\github_reptiles.py[line:43] - ERROR: 连接失败,网址为https://github.com/yangeroge/my-awesome-repo
2023-10-24 17:25:51 - D:\Hub\Reptiles_code\github_reptiles.py[line:44] - ERROR: 连接方式为get,异常信息为('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2023-10-24 17:26:17 - D:\Hub\Reptiles_code\github_reptiles.py[line:43] - ERROR: 连接失败,网址为https://github.com/yangeroge/my-awesome-repo/commits/
2023-10-24 17:26:18 - D:\Hub\Reptiles_code\github_reptiles.py[line:44] - ERROR: 连接方式为get,异常信息为HTTPSConnectionPool(host='github.com', port=443): Max retries exceeded with url: /yangeroge/my-awesome-repo/commits/ (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x000002574FE05E10>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。',))
2023-10-24 17:32:03 - D:\Hub\Reptiles_code\github_reptiles.py[line:79] - ERROR: 提交记录构成为[{'commits_auth': 'yangeroge', 'commits_time': datetime.datetime(2023, 10, 24, 11, 47, 40), 'commits_href': 'https://github.com/yangeroge/my-awesome-repo/commit/24e38eef0420e94ecd29df177c1b48dc3e22d2d3'}, {'commits_auth': 'yangeroge', 'commits_time': datetime.datetime(2023, 10, 16, 15, 11, 47), 'commits_href': 'https://github.com/yangeroge/my-awesome-repo/commit/bd4b20a6c7856490789a5b03900d56390b9c891d'}, {'commits_auth': 'yangeroge', 'commits_time': datetime.datetime(2023, 10, 10, 10, 19, 18), 'commits_href': 'https://github.com/yangeroge/my-awesome-repo/commit/0a2c12074218d7de4c9e9c1c033d0f0e778c6660'}]
2023-10-25 15:31:36 - D:\Hub\Reptiles_code\github_reptiles.py[line:81] - ERROR: 提交记录构成为[{'commits_auth': 'yangeroge', 'commits_time': '2023-10-24 11:47:40', 'commits_href': 'https://github.com/yangeroge/my-awesome-repo/commit/24e38eef0420e94ecd29df177c1b48dc3e22d2d3'}, {'commits_auth': 'yangeroge', 'commits_time': '2023-10-16 15:11:47', 'commits_href': 'https://github.com/yangeroge/my-awesome-repo/commit/bd4b20a6c7856490789a5b03900d56390b9c891d'}, {'commits_auth': 'yangeroge', 'commits_time': '2023-10-10 10:19:18', 'commits_href': 'https://github.com/yangeroge/my-awesome-repo/commit/0a2c12074218d7de4c9e9c1c033d0f0e778c6660'}]
2023-10-25 15:33:39 - D:\Hub\Reptiles_code\github_reptiles.py[line:81] - ERROR: 提交记录构成为[{'commits_auth': 'yangeroge', 'commits_time': '2023-10-24 11:47:40', 'commits_href': 'https://github.com/yangeroge/my-awesome-repo/commit/24e38eef0420e94ecd29df177c1b48dc3e22d2d3'}, {'commits_auth': 'yangeroge', 'commits_time': '2023-10-16 15:11:47', 'commits_href': 'https://github.com/yangeroge/my-awesome-repo/commit/bd4b20a6c7856490789a5b03900d56390b9c891d'}, {'commits_auth': 'yangeroge', 'commits_time': '2023-10-10 10:19:18', 'commits_href': 'https://github.com/yangeroge/my-awesome-repo/commit/0a2c12074218d7de4c9e9c1c033d0f0e778c6660'}]
2023-10-25 15:37:24 - D:\Hub\Reptiles_code\github_reptiles.py[line:44] - ERROR: 连接失败,网址为https://github.com/yangeroge/my-awesome-repo
2023-10-25 15:37:24 - D:\Hub\Reptiles_code\github_reptiles.py[line:45] - ERROR: 连接方式为get,异常信息为('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2023-10-25 15:37:55 - D:\Hub\Reptiles_code\github_reptiles.py[line:44] - ERROR: 连接失败,网址为https://github.com/yangeroge/my-awesome-repo/commits/
2023-10-25 15:37:55 - D:\Hub\Reptiles_code\github_reptiles.py[line:45] - ERROR: 连接方式为get,异常信息为HTTPSConnectionPool(host='github.com', port=443): Max retries exceeded with url: /yangeroge/my-awesome-repo/commits/ (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x000001CEFB9D6CF8>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。',))
2023-10-25 15:44:42 - D:\Hub\Reptiles_code\github_reptiles.py[line:44] - ERROR: 连接失败,网址为https://github.com/yangeroge/my-awesome-repo
2023-10-25 15:44:42 - D:\Hub\Reptiles_code\github_reptiles.py[line:45] - ERROR: 连接方式为get,异常信息为('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2023-10-25 16:42:29 - D:\Hub\Reptiles_code\github_reptiles.py[line:44] - ERROR: 连接失败,网址为https://github.com/yangeroge/my-awesome-repo/commits/
2023-10-25 16:42:29 - D:\Hub\Reptiles_code\github_reptiles.py[line:45] - ERROR: 连接方式为get,异常信息为('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2023-10-25 16:42:31 - D:\Hub\Reptiles_code\github_reptiles.py[line:88] - ERROR: 获取提交记录失败失败原因为'NoneType' object has no attribute 'text'
2023-10-26 14:14:40 - D:\Hub\Reptiles_code\github_reptiles.py[line:44] - ERROR: 连接失败,网址为https://github.com/yangeroge/code_reptiles
2023-10-26 14:14:44 - D:\Hub\Reptiles_code\github_reptiles.py[line:45] - ERROR: 连接方式为get,异常信息为('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2023-10-26 14:45:40 - D:\Hub\Reptiles_code\github_reptiles.py[line:41] - ERROR: ssh_key认证成功
2023-10-26 15:14:56 - D:\Hub\Reptiles_code\github_reptiles.py[line:41] - ERROR: ssh_key认证成功
2023-10-26 15:15:42 - D:\Hub\Reptiles_code\github_reptiles.py[line:62] - ERROR: 连接失败,网址为https://github.com/yangeroge/code_reptiles
2023-10-26 15:15:43 - D:\Hub\Reptiles_code\github_reptiles.py[line:63] - ERROR: 连接方式为get,异常信息为('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2023-10-26 15:15:44 - D:\Hub\Reptiles_code\github_reptiles.py[line:41] - ERROR: ssh_key认证成功
2023-10-26 15:16:09 - D:\Hub\Reptiles_code\github_reptiles.py[line:62] - ERROR: 连接失败,网址为https://github.com/yangeroge/code_reptiles/commits/
2023-10-26 15:20:11 - D:\Hub\Reptiles_code\github_reptiles.py[line:41] - ERROR: ssh_key认证成功
2023-10-26 15:21:21 - D:\Hub\Reptiles_code\github_reptiles.py[line:41] - ERROR: ssh_key认证成功
2023-10-26 15:26:04 - D:\Hub\Reptiles_code\github_reptiles.py[line:98] - ERROR: 提交记录构成为[{'commits_auth': 'zhukr', 'commits_time': '2023-10-26 11:31:25', 'commits_href': 'https://github.com/yangeroge/code_reptiles/commit/d7b97755210f2da3bfe003de65e9387c408bb8ff'}]
2023-10-26 15:32:40 - D:\Hub\Reptiles_code\github_reptiles.py[line:41] - ERROR: ssh_key认证成功
2023-10-26 15:32:45 - D:\Hub\Reptiles_code\github_reptiles.py[line:41] - ERROR: ssh_key认证成功
2023-10-26 15:39:57 - D:\Hub\Reptiles_code\github_reptiles.py[line:41] - ERROR: ssh_key认证成功
2023-10-26 15:40:25 - D:\Hub\Reptiles_code\github_reptiles.py[line:62] - ERROR: 连接失败,网址为https://github.com/yangeroge/code_reptiles
2023-10-26 15:40:25 - D:\Hub\Reptiles_code\github_reptiles.py[line:63] - ERROR: 连接方式为get,异常信息为('Connection aborted.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))
2023-10-26 15:40:26 - D:\Hub\Reptiles_code\github_reptiles.py[line:41] - ERROR: ssh_key认证成功
